practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/repository/HapiFhirRepository.java

```diff
@@ -56,8 +56,10 @@ public String savePractitioner(Practitioner practitioner) {
             logger.info(FhirRepo.SAVE_RESPONSE.message(),
                     fhirContextWrapper.prettyPrint(responseElement));
         }
-        return practitioner.getIdElement().getValue();
+        //return practitioner.getIdElement().getValue();
```

Comment:
revert, shouldn't be modifying practitioner-service

Reply:
Reverted practitioner-service